PRASFiles.jl/src/Results/write.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    generate_systemresult(shortfall::ShortfallResult, pras_sys::SystemModel)
+
+Generate PRASFiles.jl SystemResult from PRASCore.jl ShortfallResult  
+
+# Arguments
+
+    - `shortfall::ShortfallResult`: ShortfallResult 
+    - `pras_sys::SystemModel`: PRAS SystemModel
+
+# Returns
+
+  - SystemResult.
+"""
+function generate_systemresult(shortfall::ShortfallResult, pras_sys::SystemModel)
```

Comment:
Do we actually want to expose this function to the end-user? Seems like it's more of an internal implementation detail?